Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -0,0 +1,65 @@
+public class Game {
+	var word: String
+	let guessManager: GuessManager
+	var view: View
+	public var gameOver: Bool
+	public var winner: Bool
```

Comment:
Winner state is tracked because I need to print results outside of the playerTurn function where my guess is established. I need to know whether or not there is a winner when the game ends because I need to know which message (win/lose) to print.